confignode/src/main/java/org/apache/iotdb/confignode/partition/SchemaPartitionInfo.java

```diff
@@ -21,42 +21,64 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 public class SchemaPartitionInfo {
```

Comment:
This class should be deleted and use org.apache.iotdb.commons.partition.SchemaPartitionInfo instead.